sdk/include/aditof/camera.h

```diff
@@ -134,6 +134,17 @@ class SDK_API Camera {
     virtual Status requestFrame(Frame *frame,
                                 FrameUpdateCallback cb = nullptr) = 0;
 
+    /**
+     * @brief Scale AB image with logarithmic base 10
+     * @param frame - The frame of the camera
```

Comment:
You need to specify if it is an input or output param here. The CI build fails because of this.